db/migrations/20221206092639_create_orders_table.sql

```diff
@@ -0,0 +1,18 @@
+-- migrate:up
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT,
+    order_random_id VARCHAR(200) NOT NULL,
+    user_id INT NOT NULL,
+    status_id INT NOT NULL,
+    product_id INT NOT NULL,
+    quantity INT NOT NULL,
+    amount DECIMAL NOT NULL,
+    address VARCHAR(1000) NOT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(user_id) REFERENCES users(id),
+    FOREIGN KEY(status_id) REFERENCES statuses(id),
+    FOREIGN KEY(product_id) REFERENCES products(id)
+);
```

Comment:
random한 내용은 우리 서비스에서만 알고 있고, `order_id`로 컬럼명을 수정해도 좋을 것 같습니다.

- address 컬럼 뿐만 아니라 order_message 컬럼도 있으면 좋을 것 같네요! 해당 부분은 프론트엔드에서 배송 메시지 란을 입력하는지 확인하고 적용하면 좋을 것 같습니다!
- 외래키 설정 시 제약조건을 지정하는 방법도 참고해주세요~!

```suggestion
CREATE TABLE orders (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    order_id VARCHAR(200) NOT NULL,
    user_id INT NOT NULL,
    status_id INT NOT NULL,
    product_id INT NOT NULL,
    quantity INT NOT NULL,
    amount DECIMAL(10, 2) NOT NULL,
    address VARCHAR(1000) NOT NULL,
    order_message VARCHAR(200) NULL,
    CONSTRAINT orders_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
    CONSTRAINT orders_status_id_fkey FOREIGN KEY(status_id) REFERENCES statuses(id),
    CONSTRAINT orders_product_id_fkey FOREIGN KEY(product_id) REFERENCES products(id)
);
```